src/components/Search.js

```diff
@@ -0,0 +1,13 @@
+import './Search.css';
+
+export default function Search(){
```

Comment:
```suggestion
export default function Search() {
```

ezeket a kihagyott space-ket mindenhol nézd át kérlek